packages/namekit-react/src/components/DomainCard.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+
+export type DomainCardProps = {
+  name: string;
+};
+
+export function DomainCard({ name }: DomainCardProps) {
+  return (
+    <div>
+      Domain card for <span className="ens-webfont">{name}</span>
```

Comment:
I believe it can create too much complexity to try to separate `nameguard-react` from `namekit-react`. My assumption is that we want to merge both of these together into a single package: `namekit-react`.

If we don't do this, it gets really complicated from a dependency perspective. For example:

1. `namekit-react` needs `DisplayedName` that Franco is working on which is currently in `nameguard-react`.
2. `namekit-react` need will need the ability to show `DomainCard`, and inside `DomainCard` we need to put UI components from `nameguard-react.

My vote is to just be pragmatic and merge these packages together under the name "namekit-react". If you see it differently please let me know :+1: